tools/cli/internal/cli/merge/merge.go

```diff
@@ -29,14 +29,18 @@ const (
 	DefaultOutputFileName = "FOAS.json"
 )
 
+// writeToFileFunc is a variable that holds the function used to write files.
+// It can be overridden in tests.
+var writeToFileFunc = os.WriteFile
```

Comment:
yes, it is. I was curious to know what you think about this approach 🤣 yes, using afero seems a better approach indeed. I will update it

Reply:
use afero or use an interface, I think `io.Writer` is the common file writer interface depends how much magic you wnat since afero will give you a full wrapper on the file system which is handy

Reply:
added afero